Phonebook.Frontend/src/app/shared/dialogs/user-information/user-information.component.scss

```diff
@@ -0,0 +1,15 @@
+.center {
```

Comment:
Can we have a discussion about when something is centered and when not? Recently we are doing it differently everywhere.